doc/api/async_hooks.md

```diff
@@ -203,13 +203,16 @@ UDPSENDWRAP, UDPWRAP, WRITEWRAP, ZLIB, SSLCONNECTION, PBKDF2REQUEST,
 RANDOMBYTESREQUEST, TLSWRAP
 ```
 
+There is also the `PROMISE` resource type, which is used to track `Promise`
```

Comment:
@AndreasMadsen The reason I decided to pass the "type" (history note: using `Provider` was not the name I wanted to use, but switched long ago b/c of pressure to change on the PR) was for to allow filtering resources. I'd like to have a list of all "core types" (which would include `TickObject` and `Timer`) and I'd expect module authors to document their own "types".

Though because this relies a bit on internals could we put a note like "subject to change in future major (or minor?) releases without deprecation"?